extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -416,6 +416,20 @@ export class AmpIframe extends AMP.BaseElement {
     return super.getPriority();
   }
 
+  /** @override */
+  mutatedAttributesCallback(mutations) {
+    const src = mutations['src'];
+    if (src !== undefined) {
+      const iframeSrc = this.transformSrc_(src);
+      if (this.iframe_) {
+        this.iframe_.src = this.assertSource(
+            iframeSrc, window.location.href, this.sandbox_);
+      } else {
```

Comment:
Misread the code and thought it was set to null after load. Looks like `onload` is called again after changing the `src` anyways. Thanks!